Library/Homebrew/cmd/config.rb

```diff
@@ -131,8 +131,9 @@ def describe_system_ruby
 
   def describe_java
     unless OS.mac?
-      return "N/A" if which("java").nil?
-      java = `java -version 2>&1`.lines.first.chomp
+      return "N/A" if (which "java").nil?
+      java = `java -version 2>&1`
+      java = java.lines.first.chomp
```

Comment:
Not really... when the standalone build completed, this error was gone. I've recorded all the steps I was taking so can repeat them if someone else comes across this error and solution will be needed. 
Anyway, I will remove this hunk and the one for `gmp` + fix the glibc formula and update the request.